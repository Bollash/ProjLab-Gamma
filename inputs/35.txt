Create Space
Add Asteroid Iron
Add Settler 0
Add TpGate Asteroid 0 Settler 0
Mod Layer 0 0
Mod Core 0 Nothing
Mod Counter 1
Countdown
Status All