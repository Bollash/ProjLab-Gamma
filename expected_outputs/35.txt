Space létrehozva
A nyersanyaggal teli aszteroida hozzá lett adva a space-hez.
Sikeres volt a telepeshozzáadás
Hozzáadtuk a teleportkapukat a célpontokhoz.
Az adott aszteroida kérge a kapott értékre lett állítva.
Az aszteroida magja a kapott nyersanyagra lett állítva
A napszél visszaszámláló az adott értékre állítódott.
A számláló elérte a nullát és napszél lett. A számláló visszaállt a kiindulási állapotba.
Space
1
10
10
False
1
3
Asteroid
0
3
3
0
1
tp 1
2
0
1
Settler
0
0
1
0
TpGate
1
ast 0
2
True
TpGate
2
set 0
1
False
